libs/apollo-client/src/lib/apollo-client.ts

```diff
@@ -61,12 +64,34 @@ export function createClient(base?: string, cacheConfig?: InMemoryCacheConfig) {
     : httpLink;
 
   const errorLink = onError(({ graphQLErrors, networkError }) => {
-    if (graphQLErrors) console.log(graphQLErrors);
-    if (networkError) console.log(networkError);
+    if (graphQLErrors && !hasNotFoundGraphQLErrors(graphQLErrors)) {
+      console.log(graphQLErrors);
+    }
+    if (networkError) {
+      console.log(networkError);
+    }
   });
 
   return new ApolloClient({
     link: from([errorLink, timeoutLink, retryLink, splitLink]),
     cache: new InMemoryCache(cacheConfig),
   });
 }
+
+const isApolloGraphQLError = (
```

Comment:
question (non blocking): Why do we need this type guard?  `graphqlErrors` is already typed as `GraphqlErrors | undefined` so shouldn't a simple undefined check should suffice?

Reply:
won't compile unless `error is ApolloError`